third_party/herumi/bls_eth_go_binary.BUILD

```diff
@@ -143,8 +143,5 @@ go_library(
     }),
     copts = OPTS,
     cgo = True,
-    visibility = [
-        # Additional access will require security approval.
-        "@prysm//shared/bls:__pkg__",
```

Comment:
Please do not remove this. We can accept another whitelisted target for now, but ultimately want everything to use `//shared/bls:go_default_library`.

Reply:
Apologies; I couldn't find a way of doing this explicitly without creating a dependency loop.  With the changes made elsewhere in the patch this now works with an explicit package name.

Reply:
No worries!